dbt/adapters/default.py

```diff
@@ -89,6 +90,18 @@ def cancel_connection(cls, project, connection):
     # FUNCTIONS THAT SHOULD BE ABSTRACT
     ###
     @classmethod
+    def get_result_from_cursor(cls, cursor):
+        data = []
+
+        if cursor.description is not None:
+            column_names = [col[0] for col in cursor.description]
+            raw_results = cursor.fetchall()
+            data = [dict(zip(column_names, row))
```

Comment:
this could so readily be a `DataFrame`. Don't think we're there yet... but this is great